test/expected.json

```diff
@@ -340,21 +340,13 @@ Resources:
                   - ""
                   - - "arn:"
                     - Ref: AWS::Partition
-                    - ":logs:"
-                    - Ref: AWS::Region
-                    - ":"
-                    - Ref: AWS::AccountId
-                    - :log-group:/aws/codebuild/
+                    - :logs:us-east-1:712950704752:log-group:/aws/codebuild/
```

Comment:
Why is it less general now?

Reply:
Because `stack.account` & `stack.region` return more specific information.